changelog/fragments/upgrade-sdk-version.yaml

```diff
@@ -0,0 +1,23 @@
+# entries is a list of entries to include in
+ # release notes and/or the migration guide
+ entries:
+   - description: >
+       Updated SDK to use Go version 1.15.
+
+     # kind is one of:
+     # - addition
+     # - change
+     # - deprecation
+     # - removal
+     # - bugfix
+     kind: "change"
+
+     # Is this a breaking change?
+     breaking: true
+
+     # Migration can be defined to automatically add a section to
+     # the migration guide. This is required for breaking changes.
+     migration:
+       header: Update SDK to use go version 1.15
+       body: >
+         Operator SDK has been updated to use go version 1.15.
```

Comment:
This is a non-breaking change right?

Go 1.15 maintains the Go 1 compatibility promise, and since this PR only changes the version of Go used to compile this project, it won't have any effect on user code anyway (now that this project does not have user-facing library code).

Reply:
IHMO we ought to not add the fragment if it will not impact the end-users.

Reply:
Yeah good point! I agree.